examples/model_compress/pruning/mobilenet_end2end/test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import os
+import torch
+import torch.nn as nn
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+import numpy as np
+
+from utils import *
```

Comment:
seems we don't need to use `import *` here, we should try to avoid using this.

Reply:
fixed